backend/src/controllers/ProjectController.ts

```diff
@@ -225,6 +227,27 @@ class ProjectController {
     return projects;
   }
 
+  public async findUserProjects(userId: string): Promise<ProjectDoc[]> {
+    let userMembers = await this.memberModel.aggregate([
+      { $match: { user: new mongoose.Types.ObjectId(userId) } },
+      { $group: { _id: "$project" } },
+      { $group: { _id: null, projects: { $push: "$_id" } } },
+    ]);
+
+    userMembers = userMembers[0].projects;
+
+    let userProjects = await this.projectModel.aggregate([
+      {
+        $match: {
+          _id: { $in: userMembers },
+        },
+      },
+      ...createJoins(),
```

Comment:
Unless there's a particular reason why we wanted to add all the project team infos in the result?